doc/source/development/rfc/rfc76_ogrpythondrivers.rst

```diff
@@ -365,11 +365,11 @@ Limitations and scope
 
 - Vector and read-only for now. This could later be extended of course.
 
-- No connection between the Python code of the plugin and the OGR Python API 
```

Comment:
These whitespace changes are coming in from `.editorconfig`. If they're undesirable in commits like this it might make sense to bulk apply those settings to the `.rst` files.